src/cljc/harja/kyselyt/specql.cljc

```diff
@@ -59,9 +59,11 @@
 (defrecord NumberTransform []
   tx/Transform
   (from-sql [_ num]
-    (float num))
+    (when num
```

Comment:
Tämä taitaa olla vähä duplikaatti funkkari.
Tuolla yllä on #(when % (float %)) ja se on nimetty to-float :iksi. Voisko näistä tehdä yhden ja käyttää samaa?

Reply:
Äh, luin vain ton toisen pulleron kommentit, mutta juu, kyllä vois. Teenpä sen tuohon työn alla olevaan branchiin.